src/templates/options.js

```diff
@@ -19,3 +19,9 @@ for (const [key, value] of Object.entries(options)) {
     }
 }
 %>
+
+/* <% if (options.langDir) { %> */
+export const ASYNC_LOCALES = {
+  <%= options.locales.map(l => `'${l.code}': () => import('${options.langDir}/${l.file || l.iso}' /* webpackChunkName: "lang-${l.code}" */)`).join(',\n  ') %>
```

Comment:
Notes:

Generates this:
```js
export const ASYNC_LOCALES = {
  'en': () => import('../../lang/en' /* webpackChunkName: "lang-en" */),
  'fr': () => import('../../lang/fr-FR' /* webpackChunkName: "lang-fr" */)
}
```

Adds a new feature to fallback to iso file default filename. @rchl please let me know if you prefer to move it to a feat PR.

Reply:
Made a separate comment about the `iso` thing.